geomesa-gt/geomesa-gt-dist/pom.xml

```diff
@@ -0,0 +1,136 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <parent>
+        <groupId>org.locationtech.geomesa</groupId>
+        <artifactId>geomesa-gt_2.11</artifactId>
+        <version>2.4.0-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+    <packaging>pom</packaging>
+
+    <artifactId>geomesa-gt-dist_2.11</artifactId>
+    <name>GeoMesa GeoTools Distribution</name>
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <gmtools.dist.name>GEOMESA_GEOTOOLS</gmtools.dist.name> <!-- used in tools filtering -->
```

Comment:
Ok... that's interesting.  Does the param need to be uppercase?

Reply:
it's an environment variable I believe... we created this pattern a while ago to inject things into common scripts.